src/task.js

```diff
@@ -0,0 +1,7 @@
+export default class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed || false; // default value
+    this.index = index || -1; // default value before pushing into list
+  }
+}

```

Comment:
Try to avoid overuse classes. It is important when you need to define methods for that. I think you can manage this part directly with an array of objects.

Reply:
The suggested approach was implemented.